openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -259,3 +242,34 @@ def get_credit_provider_display_names(course_key):
         cache.set(cache_key, provider_names, credit_config.cache_ttl)
 
     return provider_names
+
+
+def make_providers_strings(providers):
+    """Get the list of course providers and make them comma seperated string.
+
+    Arguments:
+        providers : List containing the providers names
+
+    Returns:
+        strings containing providers names in readable way .
+    """
+    providers_string = None
```

Comment:
I think this function can be made more readable -- I'll put specific comments below.